### PR TITLE
Add Support for health checks with unknown protocols

### PR DESCRIPTION
**Pull request by Poltergeist**
This adds the support for healthchecks without a known protocol. We will omit those from the form

image

### CODE REVIEW

**Comment by Poltergeist** [comment]
run integration test

**Comment by Poltergeist** [comment]
Run integration tests

**Comment by Poltergeist** [comment]
Run integration tests

File: plugins/services/src/js/components/forms/HealthChecksFormSection.js [hunk 1]
```diff
@@ -171,6 +171,11 @@ class HealthChecksFormSection extends Component {
     return data.map((healthCheck, key) => {
       const errors = errorsLens.at(key, {}).get(this.props.errors);
 
+      if (!['', 'HTTP', 'HTTPS', 'COMMAND'].includes(healthCheck.protocol) &&
```
Comment (nLight) [inline plugins/services/src/js/components/forms/HealthChecksFormSection.js]: Let's extract a utility function i.e. `HealthCheckUtil.isKnownProtocol()` or extract a constant.

File: plugins/services/src/js/reducers/serviceForm/HealthChecks.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ function mapHealthChecks(item) {
       };
     }
 
-    if (item.protocol.toUpperCase().replace('HTTPS', 'HTTP') === 'HTTP') {
+    if (item.protocol.toUpperCase() !== 'COMMAND') {
```
Comment (nLight) [inline plugins/services/src/js/reducers/serviceForm/HealthChecks.js]: I would love to see protocols as constants, but this is more like a tech debt.

**Comment by Poltergeist** [comment]
The issue is now resolved @mlunoe

**Comment by mlunoe** [comment]
k, checked it out. Looks good

**Pull request merged**